src/cargo/core/workspace.rs

```diff
@@ -271,7 +271,9 @@ impl<'gctx> Workspace<'gctx> {
         ws.require_optional_deps = require_optional_deps;
         let id = package.package_id();
         let package = MaybePackage::Package(package);
-        ws.packages.packages.insert(ws.current_manifest.clone(), package);
+        ws.packages
+            .packages
+            .insert(ws.current_manifest.clone(), package);
         ws.target_dir = if let Some(dir) = target_dir {
```

Comment:
Does this reformatting belong on the previous commit?

Reply:
Oops. I squashed it into the wrong commit.